02/app.js

```diff
@@ -0,0 +1,61 @@
+const form = document.querySelector("form");
+const password1 = document.getElementById("formPass1");
+const password2 = document.getElementById("formPass2");
+const mail = document.getElementById("formLogin");
+const acceptBox = document.getElementById("formAccept");
+const submitBtn = document.querySelector("input[type=submit]");
+
+
+submitBtn.addEventListener('click', (e) => {
+    const errors = []
+  
+    if (!mail.value.includes('@')) {
+        errors.push(mail);
+    } else {
+        changeBackLabelColor(mail);
+    }
+
+    if (password1.value.length <= 6) {
+        errors.push(password1);
+    } else {
+        changeBackLabelColor(password1);
```

Comment:
Może zamiast pisać za każdym razem `else`. To może zrobić przed "zakolorowaniem na czerwono" pętle, która wyczyści (lub zakoloruje na czarno) wszystkie elementy - będzie mniej pisania, ale oba rozwiązania są ok.